src/index.js

```diff
@@ -92,7 +90,7 @@ function syncReduxAndRouter(history, store, selectRouterState = SELECT_STATE) {
 
   const unsubscribeHistory = history.listen(location => {
     const route = {
-      path: history.createPath(location),
+      path: history.createHref(location),
```

Comment:
it is needed to extract the url encoded path for current implementation. if we decided to mirror `history` location objects with query params, this line will not be necessary